doc/nrf/gs_modifying.rst

```diff
@@ -154,6 +249,17 @@ The documentation for each :ref:`configuration option <configuration_options>` a
    All changes to the :file:`.config` file are lost when you clean your build directory.
    You can save it to another location, but you must then manually copy it back to your build directory.
 
+The configuration does not change until the project is rebuilt using the ``pristine`` variable.
```

Comment:
``pristine`` is not a variable to my knowledge. It's a build target.

Reply:
Fixed.